exonum-java-binding/core/rust/Cargo.lock

```diff
@@ -173,24 +173,33 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "backtrace"
-version = "0.3.40"
+version = "0.3.38"
```

Comment:
We again face that problem with decreasing versions for some dependencies by anchoring.